### PR TITLE
Feat/event attend api

### PR DESCRIPTION
**Pull request by Aborysa**
- Add endpoints for attending/unattending events
 - Add endpoints for fetching additional event information

### CODE REVIEW

File: src/events/api/events.ts [hunk 1]
```diff
@@ -47,6 +54,33 @@ export const getAttendanceEvent = async (id: number): Promise<IAttendanceEvent>
...
+};
+
+export const getPublicAttends = async (event_id: number): Promise<IPublicAttedee[]> => {
+  try {
+    const ret: IPublicAttedee[] = await get(`${ATTENDANCE_EVENT_API_URL}${event_id}${EVENT_PUBLIC_ATTENDEES}`);
+    return ret;
+  } catch (response) {
+    throw new Error('Kunne hente påmeldingsliste!');
+  }
+};
```
Comment (oleast) [inline src/events/api/events.ts:82]: This has already been implemented, see the `publicAttendees` slice and `api/publicAttendee`

File: src/events/api/events.ts [hunk 2]
```diff
@@ -47,6 +54,33 @@ export const getAttendanceEvent = async (id: number): Promise<IAttendanceEvent>
...
 };
 
+export const getEventPayment = async (event_id: number): Promise<IPayment> => {
+  try {
+    const ret = await get<IPayment>(`${ATTENDANCE_EVENT_API_URL}${event_id}${EVENT_PAYMENT_URL}`);
+    return ret;
+  } catch (response) {
+    throw new Error('Kunne ikke hente betalingsinformasjon for arrangementet!');
+  }
+};
```
Comment (oleast) [inline src/events/api/events.ts:64]: I think this has also been implemented as part of the `payments` slice in the payments app

File: src/events/api/attendee.ts [hunk 3]
```diff
@@ -1,12 +1,63 @@
...
+}
+
+export const getAttendeeForEvent = async (eventId: number): Promise<IAttendee> => {
   const user = await getUser();
   const attendee = await get<IAttendee>(getEventAttendeeUrl(eventId), { format: 'json' }, { user });
   return attendee;
 };
+
+export const userAttendEvent = async (
+  event_id: number,
```
Comment (oleast) [inline src/events/api/attendee.ts:25]: I think we should use camel casing until we get at far down to the API as possible

--
Maybe we should automatically transform all snake casing for requests at a later time?

**Comment by FluidSense** [comment]
Will this close #351?
If it will, can it be set to auto-close it? 😄

**Comment by oleast** [comment]
Yes it will, @FluidSense 😁

**Pull request merged**